### ISSUE TITLE
Image field in grid mode should not crop the image in the popup

### ISSUE THREAD

**Issue by vistamedia** [FLEXIcontent/flexicontent-cck#783]
To be more clear, the parameter zc=1 should be taken off the url of the zoomed image.

![zoom-crop-1](https://user-images.githubusercontent.com/9335926/30740643-1d018104-9f92-11e7-8a16-7aafbeefe0f7.jpg)
![zoom-crop-2](https://user-images.githubusercontent.com/9335926/30740642-1d00f46e-9f92-11e7-8365-523ceb87c4dd.jpg)

**Comment by ggppdk** [comment]
Yes i confirm this, the zoomed image is indeed created with cropping option ON

i am thinking of a button to switch the file list view to uncropped thumbnails
